### PR TITLE
Clarified hyperloglog documenation based on recent user feedback

### PR DESCRIPTION
**Pull request by jerryxwu**
# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [x] Is the content free from typos?
- [x] Does the content use plain English?
- [x] Does the content contain clear sections for concepts, tasks, and references?
- [x] Are procedure and highlight tags used appropriately?
- [x] Has the index been updated appropriately?
- [x] Have any images been uploaded to the correct location, and are resolvable?
- [x] Are all links provided in reference style, and resolvable?
- [x] If the page index was updated, are redirects required
      and have they been implemented?
- [x] Have you checked the built version of this content?

### CODE REVIEW

**Review by Loquacity — APPROVED** [review]
Approved once Vale passes.

File: api/hyperloglog.md [hunk 1]
```diff
@@ -24,13 +24,14 @@ For more information about approximate count distinct functions, see the
...
 |value|AnyElement| Column to count distinct elements. The type must have an extended, 64-bit, hash function.|
 
 Increasing the `buckets` argument usually provides more accuracy at the expense
-of more storage. Because hyperloglog is a probabilistic algorithm, it works
-best on datasets that have many distinct values: at least tens of thousands. But it should
-also be fairly accurate so long as you have more buckets than distinct values.
+of more memory usage. Because hyperloglog is a probabilistic algorithm, it works
+best on datasets that have many distinct values: at least tens of thousands. 
+
+See [stderror][stderror] for how estimated error rate is related to `buckets`.
```
Comment (charislam) [inline api/hyperloglog.md:34]: Not linked yet! Just add this to the bottom of the page:

```
[stderror]: /api/:currentVersion:/hyperfunctions/approx_count_distincts/stderror/
```

**Review by charislam — CHANGES_REQUESTED** [review]
Just one link that needs to be added :)

btw we changed the way we do our link checking, which is going to catch a lot more broken links, but during the transition it also flags a lot of false positives. If you don't have admin rights on this repo, just ping me when you're ready to merge and I'll handle it!

File: api/hyperloglog.md (hunk 1 above)
Comment (jerryxwu) [reply api/hyperloglog.md:34]: Good catch. Added.

Comment (jerryxwu) [reply api/hyperloglog.md:34]: btw, I also corrected other two links where the leading slash (/) was missing.

File: api/hyperloglog.md [hunk 2]
```diff
@@ -58,3 +59,5 @@ CREATE VIEW hll AS SELECT hyperloglog(32768, data) FROM samples;
 
 
 [hyperfunctions-approx-count-distincts]: /timescaledb/:currentVersion:/how-to-guides/hyperfunctions/approx-count-distincts/
+
```
Comment (charislam) [inline api/hyperloglog.md:62]:
```suggestion
```

**Review by charislam — APPROVED** [review]

**Pull request merged**